clayui.com/content/docs/components/markup-aspect-ratio.md

```diff
@@ -0,0 +1,623 @@
+---
+title: 'Aspect Ratio'
+---
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+-   [Aspect Ratio](#css-aspect-ratio)
+    -   [Bg Contain](#css-aspect-ratio-bg-contain)
+    -   [Bg Cover](#css-aspect-ratio-bg-cover)
+    -   [Bg Center](#css-aspect-ratio-bg-center)
+-   [Aspect Ratio Item](#css-aspect-ratio-item)
+    -   [Fluid](#css-aspect-ratio-item-fluid)
+    -   [Vertical Fluid](#css-aspect-ratio-item-vertical-fluid)
+    -   [Flush](#css-aspect-ratio-item-flush)
+    -   [Vertical Flush](#css-aspect-ratio-item-vertical-flush)
+    -   [Center Middle](#css-aspect-ratio-item-center-middle)
+    -   [Top Center](#css-aspect-ratio-item-top-center)
+    -   [Top Right](#css-aspect-ratio-item-top-right)
+    -   [Right Middle](#css-aspect-ratio-item-right-middle)
+    -   [Bottom Right](#css-aspect-ratio-item-bottom-right)
+    -   [Bottom Center](#css-aspect-ratio-item-bottom-center)
+    -   [Bottom Left](#css-aspect-ratio-item-bottom-left)
+    -   [Left Middle](#css-aspect-ratio-item-left-middle)
+
+</div>
+</div>
+
+## Aspect Ratio(#css-aspect-ratio)
+
+Sometimes you can't control the size of an image, you can constrain your images with aspect-ratios. The base class `aspect-ratio` maintains a 1 to 1 ratio relative to its container's width.
+
+Use `aspect-ratio-3-to-2`, `aspect-ratio-4-to-3`, `aspect-ratio-8-to-3`, `aspect-ratio-8-to-5`, or `aspect-ratio-16-to-9` to maintain the specific ratio relative to its container or create your own by setting `padding-bottom` to the ratio you want e.g. 2 to 1 `.aspect-ratio-2-to-1 { padding-bottom: 50% }`.
+
+<div class="sheet-example">
+	<div class="row">
+		<div class="col-3">
+			<h6>1:1</h6>
+			<div class="aspect-ratio bg-light"></div>
+		</div>
+		<div class="col-3">
+			<h6>3:2</h6>
+			<div class="aspect-ratio aspect-ratio-3-to-2 bg-light"></div>
+		</div>
+		<div class="col-3">
+			<h6>4:3</h6>
+			<div class="aspect-ratio aspect-ratio-4-to-3 bg-light"></div>
+		</div>
+		<div class="col-3">
+			<h6>8:3</h6>
+			<div class="aspect-ratio aspect-ratio-8-to-3 bg-light"></div>
+		</div>
+		<div class="col-3">
+			<h6>8:5</h6>
+			<div class="aspect-ratio aspect-ratio-8-to-5 bg-light"></div>
+		</div>
+		<div class="col-3">
+			<h6>16:9</h6>
+			<div class="aspect-ratio aspect-ratio-16-to-9 bg-light"></div>
+		</div>
+	</div>
+</div>
+
+```html
+<div class="aspect-ratio"></div>
+<div class="aspect-ratio aspect-ratio-3-to-2"></div>
+<div class="aspect-ratio aspect-ratio-4-to-3"></div>
+<div class="aspect-ratio aspect-ratio-8-to-3"></div>
+<div class="aspect-ratio aspect-ratio-8-to-5"></div>
+<div class="aspect-ratio aspect-ratio-16-to-9"></div>
+```
+
+### Aspect Ratio Bg Contain(#css-aspect-ratio-bg-contain)
```

Comment:
Hey @pat270 apparently, there was a problem with the markup of this document, for some reason from here it is rendering the sessions nested as if they were inside an example, this could be because some tags are not closed correctly.

Reply:
Thanks @matuzalemsteles! I forgot to add `</div>` on all the background examples.

Reply:
Good, thanks Patrick